lib/has_array_of/builders.rb

```diff
@@ -14,7 +14,7 @@ def has_array_of(name, options = {})
         class_name = (options[:class_name] || singular_name.camelize).to_s
         ids_attribute = "#{singular_name}_ids".to_sym
         model = class_name.constantize
-        pkey_attribute = model.primary_key.to_sym
+        pkey_attribute = model.primary_key.to_s.to_sym
```

Comment:
why?

Reply:
when you drop, create and then migrate DB, like new setup, then when rails loads code, db doesn't have this primary key and migration is failing

Reply:
so `pkey_attribute` will be `:""`?

Reply:
yes, but this will appear only during this situation. In real rails app run, never happens

Reply:
Then I would expect to have a spec for the case when no table is available for the model. At least `expect { ... }.not_to raise_error`.

BTW How ActiveRecord solves such problems? I guess it has something like lazy initialization so it will fail only when model actually been tried to use. I think we should do this but for now at least it's worth to add a `# TODO` comment for this.

Reply:
And I don't agree that it never happens in the real rails app.
Imagine the case when someone deployed a code that adds a `Pillow` model and `has_array_of :pillows` association to the `Couch` model. Code has been deployed but migration creating a `pillows` table is not yet run. Then freshly deployed app will fail to start which is serious problem.